templates/user/auth/forgot_passwd.tmpl

```diff
@@ -22,7 +22,13 @@
 							<button class="ui blue button">{{.i18n.Tr "auth.send_reset_mail"}}</button>
 						</div>
 					{{else if .IsResetDisable}}
-						<p class="center">{{.i18n.Tr "auth.disable_forgot_password_mail"}}</p>
+						<p class="center">
+							{{if $.IsAdmin}}
```

Comment:
will this ever be true? Wouldn't IsAdmin require the user to be logged in?

Reply:
Yes, and there is a "forgot password" link in user settings that would take them to this page.
Not sure what scenario it covers, exactly, though. Maybe forgot password but logged in with oauth?